src/plugins/intel_cpu/src/emitters/snippets/x64/utils.hpp

```diff
@@ -20,6 +23,44 @@ inline static std::vector<size_t> transform_snippets_regs_to_idxs(const std::vec
     return idxs;
 }
 
+/**
+ * @brief If the passed `port` is connected to a Buffer, return its cluster ID.
+ *        Otherwise returns SIZE_MAX
+ * @param port expression port of memory access op
+ * @return cluster ID of the connected Buffer or SIZE_MAX
+ */
+size_t get_buffer_cluster_id(const ov::snippets::lowered::ExpressionPort& port);
+
+/**
+ * @brief Find the available register from the pool excepting: abi_param1, RSP, RBP and `used_gpr_idxs`
+ * @param used_gpr_idxs current used gpr register indexes
+ * @return register
+ */
+Xbyak::Reg64 get_aux_gpr(const std::vector<size_t>& used_gpr_idxs);
+
+/**
+ * @brief Push data pointer on stack adding offset. The offset is gotting from runtime params `abi_param1`
```

Comment:
Minor: you can consider another formulation
```suggestion
 * @brief Push data pointer on stack adding offset. The offset is taken from runtime params `abi_param1`
```